app/assets/javascripts/admin/api_flow_types.js

```diff
@@ -2,13 +2,13 @@ import type { Vector3 } from "oxalis/constants";
 import type { DataLayerType } from "oxalis/store";
 
 type APIDataSourceType = {
-  id: {
-    name: string,
-    team: string,
+  +id: {
```

Comment:
Well, I think when I started I thought about modifying some params, so I didn't mark them as covariant. Later I realized I shouldn't be doing any in-place modifications and never got back to marking the types as covariant.